services/queue/src/api.js

```diff
@@ -192,6 +192,86 @@ builder.declare({
   return res.reply(definition);
 });
 
+/** Get task batched */
+builder.declare({
+  method: 'post',
+  route: '/tasks',
+  query: paginateResults.query,
+  name: 'tasks',
+  scopes: { AllOf: [{
+    for: 'taskId',
+    in: 'taskIds',
+    each: 'queue:get-task:<taskId>',
+  }] },
+  input: 'tasks-request.yml',
+  stability: APIBuilder.stability.experimental,
+  category: 'Tasks',
+  idempotent: true,
+  output: 'tasks-response.yml',
+  title: 'Get multiple task definitions',
+  description: [
+    'This end-point will return the task definition for each input task id.',
+    'Notice that the task definitions may have been modified by queue.',
+    'If an optional property is not specified the queue may provide',
```

Comment:
```suggestion
```

this was probably lost long time ago, as I don't see what optional property is being passed :) let's drop it